samples/highcharts/demo/polar-advanced/demo.html

```diff
@@ -0,0 +1,104 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/modules/coloraxis.js"></script>
+<script src="https://code.highcharts.com/modules/pattern-fill.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/export-data.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<!-- <script src="https://code.highcharts.com/themes/brand-light.js"></script> -->
```

Comment:
I support removing theme because there's an option to view the demos with selected themes for the users making default and the used theme (brand light) the same.
![image](https://github.com/highcharts/highcharts/assets/8960408/a88d0676-0066-4e97-8092-dcc0e9bafa50)